### PR TITLE
PIN Layout viewing options added

### LINKED ISSUE #397: PIN layout options in launcher screen
**Actual Behaviour.**  
Missing PIN layout options in launcher screen.

**Expected Behaviour**  
PIN layout options in launcher screen.

**Steps to reproduce it**  
> Not available

**Additional context**  
> Not available

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #397 

**Changes**: PIN Layout viewing options added

**Screenshot/s for the changes**: 

 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
https://drive.google.com/open?id=1UtOvn1PiFrYS9NvPZDRGyF0QtBWj_Iss

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -22,7 +21,8 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".activities.MeditationListActivity"></activity>
+        <activity android:name=".activities.PinLayoutActivity"></activity>
```
Comment (CloudyPadmal) [inline app/src/main/AndroidManifest.xml]: `<activity android:name=".activities.PinLayoutActivity" />`

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Rotate the images to have the screen filled

**Comment by CloudyPadmal** [comment]
Images are not rotated yet

**Comment by jddeep** [comment]
@CloudyPadmal

**Review by CloudyPadmal — APPROVED** [review]

**Pull request merged**